src/org/zaproxy/zap/extension/AllInOneNotes/ExtensionAllInOneNotes.java

```diff
@@ -0,0 +1,211 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ * 
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); 
+ * you may not use this file except in compliance with the License. 
+ * You may obtain a copy of the License at 
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0 
+ *   
+ * Unless required by applicable law or agreed to in writing, software 
+ * distributed under the License is distributed on an "AS IS" BASIS, 
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+ * See the License for the specific language governing permissions and 
+ * limitations under the License. 
+ */
+package org.zaproxy.zap.extension.AllInOneNotes;
+
+import java.awt.CardLayout;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.ArrayList;
+
+import javax.swing.*;
```

Comment:
I'm sure this is an IDE thing (likely IntelliJ?) but specific imports should be used.

Reply:
yep you're completely right. changed IntelliJ to modify this behavior. Here's the solution in case anyone's interested:

https://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports